cli/src/main.rs

```diff
@@ -126,8 +127,10 @@ struct EthereumArgs {
     rpc_port: Option<u16>,
     #[clap(short = 'w', long, env)]
     checkpoint: Option<B256>,
-    #[clap(short, long, env, value_parser = parse_url)]
+    #[clap(long, env, value_parser = parse_url)]
```

Comment:
Lets keep the shorthand for this. It's likely the one people would want to set themselves, since its unlikely there will be other hosts of the verifiable api for some time, so users will likely stick to the default there.

Reply:
Oh yes, I made this change to resolve some clippy issue but I think it can be reverted.

Reply:
```bash
Command ethereum: Short option names must be unique for each argument, but '-e' is in use by both 'execution_rpc' and 'execution_verifiable_api'
```

I had removed it because of this error. We can keep `-e` for `execution_rpc` and provide a new shorthand like `-ev` for `execution_verifiable_api`?